rpcserver.go

```diff
@@ -302,24 +302,50 @@ func (r *rpcServer) DebugLevel(ctx context.Context,
 func (r *rpcServer) MintAsset(ctx context.Context,
 	req *tarorpc.MintAssetRequest) (*tarorpc.MintAssetResponse, error) {
 
+	// A request should be a new asset or a command for the planter.
+	if req.Asset == nil && !req.FinalizeBatch {
+		return nil, fmt.Errorf("asset is required")
+	}
+
+	// Finalizing a batch implies not adding another asset to the batch.
+	if req.FinalizeBatch && req.Asset != nil {
+		return nil, fmt.Errorf("cannot finalize batch and add asset")
+	}
+
+	if req.FinalizeBatch {
```

Comment:
Yeah, I was thinking that these 'batch commands' should really be separate on the RPC and CLI - just realized that we can add actual subcommands to the 'mint' subcommand instead of using flags.

So I can then have separate functions in `rpcserver` and add new endpoints + minting subcommands.